includes/Core/Util/User_Input_Settings.php

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * Class Google\Site_Kit\Core\Util\User_Input_Settings
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Authentication\Authentication;
+use Google\Site_Kit\Core\Authentication\Google_Proxy;
+use Google\Site_Kit\Core\Storage\Transients;
+use Google\Site_Kit\Core\Storage\User_Transients;
+
+/**
+ * Class managing requests to user input settings endpoint.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class User_Input_Settings {
+
+	/**
+	 * Authentication object.
```

Comment:
Please use "instance" for consistency with other classes.
```suggestion
	 * Authentication instance.
```

Same below.

Reply:
Updated.